ide/urls.py

```diff
@@ -14,9 +17,15 @@
     url(r'^keras/', include('keras_app.urls')),
     url(r'^tensorflow/', include('tensorflow_app.urls')),
     url(r'^save$', save_to_db, name='saveDB'),
+    url(r'^saveModel$', save_model_to_db, name='saveModel'),
     url(r'^load*', load_from_db, name='loadDB'),
+    url(r'^deleteModel$', delete_model_from_db, name='deleteModel'),
+    url(r'^getModel$', load_model_from_db, name='getModelData'),
     url(r'^model_history', fetch_model_history, name='model-history'),
-    url(r'^model_parameter/', calculate_parameter, name='calculate-parameter'),
-    url(r'^layer_parameter/', fetch_layer_shape, name='fetch-layer-shape')
-] + static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT) + \
-    static(settings.STATIC_URL, document_root=settings.STATIC_ROOT)
+    url(r'^model_parameter/', calculate_parameter,
+        name='calculate-parameter'),
+    url(r'^layer_parameter/', fetch_layer_shape,
+        name='fetch-layer-shape'),
+    ] + static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT) \
```

Comment:
Fix alignment

Reply:
This was according to the pep8. It gives an error otherwise.